api/src/routers/meals.js

```diff
@@ -0,0 +1,71 @@
+import express from "express";
+import knex from "../database_client.js";
+
+const router = express.Router();
+
+// GET /api/meals - Returns all meals
+router.get("/", async (req, res) => {
+  try {
+    const meals = await knex.select("*").from("meals");
+    res.json(meals);
+  } catch (error) {
+    res.status(500).json({ error: error.message });
```

Comment:
check here: https://expressjs.com/en/guide/error-handling.html

see if you can implement a error handling via middleware